ee/clickhouse/queries/trends/total_volume.py

```diff
@@ -61,23 +58,45 @@ def _total_volume_query(self, entity: Entity, filter: Filter, team_id: int) -> T
                     **content_sql_params,
                     parsed_date_to=trend_event_query.parsed_date_to,
                     parsed_date_from=trend_event_query.parsed_date_from,
-                    **trend_event_query.active_user_params
+                    **trend_event_query.active_user_params,
                 )
             else:
                 content_sql = VOLUME_SQL.format(event_query=event_query, **content_sql_params)
 
             null_sql = NULL_SQL.format(trunc_func=trunc_func, interval_func=interval_func)
             params["interval"] = filter.interval
             final_query = AGGREGATE_SQL.format(null_sql=null_sql, content_sql=content_sql)
-            return final_query, params, self._parse_total_volume_result(filter)
+            return final_query, params, self._parse_total_volume_result(filter, entity, team_id)
 
-    def _parse_total_volume_result(self, filter: Filter) -> Callable:
+    def _parse_total_volume_result(self, filter: Filter, entity: Entity, team_id: int) -> Callable:
         def _parse(result: List) -> List:
             parsed_results = []
             for _, stats in enumerate(result):
                 parsed_result = parse_response(stats, filter)
+                parsed_result.update(
+                    {"persons_urls": self._get_persons_url(filter, entity, team_id, parsed_result["days"])}
+                )
                 parsed_results.append(parsed_result)
 
             return parsed_results
 
         return _parse
+
+    def _get_persons_url(self, filter: Filter, entity: Entity, team_id: int, dates: List[str]) -> List[Dict[str, Any]]:
+        persons_url = []
+        for date in dates:
+            filter_params = filter.to_dict()
+            extra_params = {
+                "entity_id": entity.id,
+                "entity_type": entity.type,
+                "date_from": date,
+                "date_to": date,
```

Comment:
Is this right only for day interval or will this all come out in the wash somehow?

Perhaps we can, as part of the results SELECT also include both lower and upper date bounds to be completely explicit? Optional, though if it already works as is.

Reply:
I was following the format that's been laid out and currently for any interval it works with date_from/date_to sync'd like above

Reply:
Cool, does this include things like "all time" filtering ?

Reply:
nope i need to add the test coverage for these edges